dev.md

```diff
@@ -39,15 +39,42 @@ There are 2 methods to develop for Wiki.js. You can either use the dockerized de
 9. Browse to **http://localhost:3000/** _(replace localhost with the hostname of your machine if applicable)_.
 10. Complete the setup wizard to finish the installation.
 
-## Stopping the project
+### Stopping the project
 
 Click on **File > Close Remote Connection** to stop the containers and close the Visual Studio Code instance.
 
-## Removing the containers
+### Removing the containers
+
+When you're done and no longer need the development environment, open the **Remote Explorer** tab and remove all containers starting with the name `wiki`.
 
-When you're done and no longer need the development environment, open the **Remote Explorer** tab and remove all containers starting with the name `containers`.
+Alternatively, see the [generic method](#generic-removing-the-containers) below.
 
-An alternate method is to open a command prompt in the `dev/containers` folder and run `docker-compose down`.
+## Generic
+
+### Running the project
+1. Clone the project from [GitHub](https://github.com/Requarks/wiki).
+2. Run the following commands:
+```
+docker-compose -f dev/containers/docker-compose.yml up -d
+docker exec wiki-app yarn   # only necessary the first time (todo: why is it necessary at all?)
```

Comment:
In order to install the npm dependencies.

Reply:
Ah got it, I see that step now under "Manual". I guess VS Code takes care of this step automatically?

Reply:
Correct, see https://github.com/Requarks/wiki/blob/2fe2e6210da79003ac229027dfb06a9f992f47fd/.devcontainer/devcontainer.json#L32